datastore/datastore_test.go

```diff
@@ -50,6 +50,8 @@ type KeyValue struct {
 	Value string
 }
 
+var keyValueCache Cache // KeyValue cache.
```

Comment:
I'd argue it's better to keep values you need for each test local to the test i.e create this where you need it; I think it's best to avoid globals where you can. Consider using sub tests if you want to perform multiple tests on the same cache.

Reply:
While KeyValue is just an entity used for testing, in general, I think the cache for each entity will need to be a global, albeit limited to file scope.

Reply:
Yeah, understood.

Reply:
I still don't think it should be global in testing though; unit tests should not have the capacity to pollute other unit tests through a global. Unit tests should be self contained.